src/main/java/com/example/demo/model/dto/UserCreation.java

```diff
@@ -28,9 +31,10 @@ public String getPassword() {
 
     public void setAge(int age) {
         if (age < 0) {
-            return;
+            log.error("age validation error");
```

Comment:
If something goes wrong, you need to throw an runtime exception with the same message as in your logger. In that case you need to return Boolean. If you receive smth from the function it means it is success, otherwise spring will handle your exception and return it to the user directly